api/src/routing.js

```diff
@@ -655,6 +655,14 @@ app.all(
   dbDocHandler.request,
   authorization.setAuthorized // adds the `authorized` flag to the `req` object, so it passes the firewall
 );
+const initialReplication = require('./controllers/initial-replication');
+app.get(
+  '/initial-replication/get-ids',
+  authorization.handleAuthErrors,
+  authorization.onlineUserPassThrough,
+  jsonQueryParser,
```

Comment:
What are we parsing here?

Reply:
Oh, thanks for noticing. Initially I was passing some info, but that's now been removed.